katana/wui/core/static/core/js/main.js

```diff
@@ -1242,4 +1242,67 @@ var katana = {
 
   },
 
+    utils: {
+
+        getRelativeFilepath: function (basePath, path) {
```

Comment:
Trying the following code in chrome dev console
`katana.utils.getRelativeFilepath("/home/ka/Desktop/cases/", "/home/ka/Desktop/cases/warriorframework/katana");
"warriorframework/katana"`
`katana.utils.getRelativeFilepath("/home/ka/Desktop/cases", "/home/ka/Desktop/cases/warriorframework/katana");
"../../../..//home/ka/Desktop/cases/warriorframework/katana"`
The second example isn't expected

Reply:
Addressed.